### PR TITLE
Remove namespace from mount_point label.

### PR DESCRIPTION
**Pull request by mgritter**
This makes mount_point consistent across metrics; previously some had the namespace label and some did not:

image

### CODE REVIEW

**Comment by mgritter** [comment]
Should go in 1.5 GA, but not necessary for RC.

**Comment by ncabatoff** [comment]
We don't need to apply this fix to the other metrics that specify a mount_point, e.g. wrapInCubbyhole?

**Comment by mgritter** [comment]
> We don't need to apply this fix to the other metrics that specify a mount_point, e.g. wrapInCubbyhole?

No, you're right.  Should have moved everything to one place like you originally suggested.  :)

**Comment by mgritter** [comment]
Test on enterprise: 
image

auth/github/, prefix/mysecret/, and auth/myauth/ are in ns1
auth/myauth/ (same name) and secret/ are in root

File: vault/token_store.go [hunk 1]
```diff
@@ -1822,7 +1822,7 @@ func (ts *TokenStore) handleTidy(ctx context.Context, req *logical.Request, data
 				for index, child := range children {
 					countParentList++
 					if countParentList%500 == 0 {
-						percentComplete := float64(index)/float64(len(children))*100
```
Comment (mgritter) [inline vault/token_store.go:1825]: Oops, somebody's not using gofmt I guess?

**Pull request merged**